database/migrations/2021_02_02_175500_create_movies_table.php

```diff
@@ -31,9 +31,13 @@ public function up()
             $table->timestamps();
         });
 
-        Artisan::call('db:seed', [
-            '--class' => MoviesTableSeeder::class,
-        ]);
+        // Artisan::call('db:seed', [
```

Comment:
Should this be in comments?

Reply:
Om personen som använder detta använder windows xamp, ampps o.s.v kan vara ett problem då artisan har ingen "use ...." och du får en error. Använder du vagrant är inget problem, men website ska var anpassad till allt. 

Därför ska ni inte ta bort heller kommentarer jag skriver som grönt för att de är viktiga. Om vi skulle göra en projekt till någon den personen skulle få ett stor problem med att få php artisan migrate att fungera då Artisan::calll fungerar inte utan en rutan från "use ....." därför jag kommenterade hur man kan lägga den manuellt.